src/libtexture/imagecache_pvt.h

```diff
@@ -1197,7 +1205,7 @@ class ImageCacheImpl final : public ImageCache {
     spin_mutex m_fingerprints_mutex;  ///< Protect m_fingerprints
     FingerprintMap m_fingerprints;    ///< Map fingerprints to files
 
-    TileCache m_tilecache;          ///< Our in-memory tile cache
+    mutable TileCache m_tilecache;  ///< Our in-memory tile cache
```

Comment:
Why this change?

Reply:
@lgritz I did that change to be able to loop through TileCache from a const reference to ImageCacheImpl, because unordered_map_concurrent doesn't implement const iterators. Maybe not the best choice. But I think that's already why the FilenameMap is mutable.